m4/ac_path_kernel_source.m4

```diff
@@ -30,6 +30,15 @@ AC_DEFUN([AC_PATH_KERNEL_SOURCE_SEARCH],
         break
       fi;
     done
+    if test "$kerneldir" = /lib/modules/${vers}/build/ -a \
+      -r "$kerneldir/Makefile" -a \
+      `wc -l <"$kerneldir/Makefile"` -lt 100 -a \
+      -r "/lib/modules/${vers}/source/Makefile"
+    then
+      kerneldir_source="/lib/modules/${vers}/source"
+    else
+      kerneldir_source="$kerneldir"
+    fi
```

Comment:
seems like a hack
maybe need to push down a custom path to the kernel build dir from rpm build command?

Reply:
It's less of a hack if two distributions (Debian and SUSE) use this convention. A custom path: custom extra path? You can't use just a single path.